genapi/src/parser/integer.rs

```diff
@@ -45,7 +45,7 @@ impl Parse for IntegerNode {
         let inc = node
             .parse_if(INC, node_builder, value_builder, cache_builder)
             .or_else(|| node.parse_if(P_INC, node_builder, value_builder, cache_builder))
-            .unwrap_or(ImmOrPNode::Imm(10));
+            .unwrap_or(ImmOrPNode::Imm(1));
```

Comment:
I _believe_ this is correct, but is there somewhere in the spec which states this?

Reply:
> I _believe_ this is correct, but is there somewhere in the spec which states this?

https://www.emva.org/wp-content/uploads/GenICam_Standard_v2_1_1.pdf
Page 51. There are the defaults stated.